runtime/README.md

```diff
@@ -1,41 +1,158 @@
+### Bref Runtime Structure
 
-- `bref/tmp/cleaned-build-php-XX`
-  
-    There is one image per PHP version. These images contain PHP compiled with all its extensions,
-    but with all extra files removed. These images do not contain the bootstrap files and PHP config files.
+This project is designed for easily discarding old PHP versions and easily creating new ones. Almost everything
+for a specific PHP Version stays inside `phpxx` folder. They are also optimized for packing single layers
+for testing. Any contributor that wants to make changes to the layers will feel more confident about their
+contribution if they can test them. A layer can be placed onto your own AWS Account by executing the following
+steps:
 
-- `bref/php-XX`, `bref/php-XX-fpm`
-  
-    There is one image per PHP version. These images contain exactly what will be in the layers in `/opt`.
+- Edit `phpxx/Makefile` and add your own `AWS_PROFILE` at the top of the file.
+- Run `make test` if you want to make sure the layers check are up-to-date.
+- Run `make function` to deploy a Layer on your own account.
+- You may choose which region layers get published at the top of `phpxx/Makefile`
+- Run `make fpm` if you want to deploy an FPM Layer on your own account
+- TODO: Console Layer.
 
-    These images have 3 goals:
-  
-    - export `/opt` to create layers
-    - run applications locally using Docker: this requires overloading the entrypoint if necessary
-    - deploy applications on Lambda using Docker images: set the handler as `cmd`
+### runtime/common/function & runtime/common/fpm
 
-- `bref/php-XX-fpm-dev`
+In order to make the Runtime self-sufficient, we need to include some basic functionality from Bref inside
+the layer itself. We do this by packaging the `src` folder as a composer installation inside the `runtime/common`
+folders. Two Docker Images are generated with Bref and it's minimal dependencies. This is what allows users
+to create Lambda Functions using the AWS Console and run their function on-the-fly (no composer or serverless deploy necessary).
 
-    Used to run web applications locally.
+### runtime/common/publish
 
-Layers are created in the `export/` directory (we zip the `/opt` directory of each Docker image).
-We upload and publish the layers in every AWS region using the zip files.
+In order to publish layers in parallel, we use a `docker-compose` file with one container per region. That means
+we can upload 21 layers in bulk. The containers expect a `TYPE` environment variable to locate and upload the zip
+file. The zip files are generated and stored in `/tmp/bref-zip/` folder on the root machine.
+
+### The php{xx} folder
+
+This is the heart of Bref Runtime Layers. We configure php using the `config` folder to store `php.ini` files that
+will be copied into `/opt/php-ini`. Note that we configure `/opt/bootstrap` to execute PHP with `PHP_INI_SCAN_DIR=/opt/php-ini:/var/task/php/conf.d/`.
```

Comment:
The directory seems to have changed compared to master right?

I think we need to keep loading `ini` from the same directories (IIRC it was `/opt/bref/etc/php/conf.d/`), I don't think we gain anything by changing and that avoids BC break (e.g. for https://github.com/brefphp/extra-php-extensions or any custom layer already existing)? (was there a reason to change?)